client/consensus/api/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+// our error-chain could potentially blow up otherwise
+#![recursion_limit="128"]
+
+#[macro_use] extern crate log;
```

Comment:
hmm, is this really neccessary?

Reply:
Have removed it. Let's see if the companion build complains...

Reply:
Missing license header.